tests/CMakeLists.txt

```diff
@@ -23,9 +23,14 @@ if(BUILD_GENIVI)
     list(APPEND TEST_LIBS rvi)
 endif(BUILD_GENIVI)
 
+add_custom_target(prepare-ostree-sysroot
+    COMMAND if [ ! -d "sysroot" ]; then curl https://s3.eu-central-1.amazonaws.com/pw-dropbox/ostree-sysroot.tar.gz -o ostree-sysroot.tar.gz && tar -xf ostree-sysroot.tar.gz \; fi
+    DEPENDS aktualizr)
```

Comment:
It feels odd that we need a OSTree sysroot in order to test provisioning.  I know that the config validity checks expect to be able to find a ostree sysroot, but wouldn't a better solution be to move those checks to later in the process?  This would decouple the OSTree parts from the provisioning parts, rather than try meet the dependency?